apps/estuary/test/unit/estuary_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule EstuaryTest do
```

Comment:
I think we can safely delete this module/file at this point.

Reply:
fixed